addons/website/views/snippets.xml

```diff
@@ -528,7 +602,7 @@
 </template>
 
 <template id="s_parallax" name="Parallax">
-    <section class="parallax s_parallax oe_custom_bg oe_img_bg"
+    <section class="parallax s_parallax oe_img_bg"
```

Comment:
Yes because oe_custom_bg is for customly selected images (selected from the image dialog box) and by default, this snippet is having background selected from the available images' dropdown (cubes) and not custom one.